apps/meteor/client/components/UserStatusMenu.tsx

```diff
@@ -1,17 +1,33 @@
+import { UserStatus as UserStatusType } from '@rocket.chat/core-typings';
 import { Button, PositionAnimated, Options, useCursor, Box } from '@rocket.chat/fuselage';
+import type { Placements } from '@rocket.chat/fuselage-hooks';
 import React, { useRef, useCallback, useState, useMemo, useEffect } from 'react';
 
 import { useSetting } from '../contexts/SettingsContext';
 import { useTranslation } from '../contexts/TranslationContext';
 import { UserStatus } from './UserStatus';
 
-const UserStatusMenu = ({ onChange, optionWidth = undefined, initialStatus = 'offline', placement = 'bottom-end', ...props }) => {
+type UserStatusMenuProps = {
+	margin: string;
+	onChange: (type: UserStatusType) => void;
+	initialStatus?: UserStatusType;
+	optionWidth?: any;
+	placement?: Placements;
+};
+
+const UserStatusMenu = ({
+	margin,
+	onChange,
+	initialStatus = UserStatusType.OFFLINE,
+	optionWidth = undefined,
+	placement = 'bottom-end',
+}: UserStatusMenuProps): React.ReactElement => {
```

Comment:
```suggestion
}: UserStatusMenuProps): ReactElement => {
```

It's better to import `ReactElement` from react instead of using the namespace.